newsletters/admin.py

```diff
@@ -1,5 +1,9 @@
 from django.contrib import admin
 from .models import Newsletter
 
+class NewsletterAdmin(admin.ModelAdmin):
+    fields = ('title','body','author','pub_date','summary','ispublished','banner')
```

Comment:
This errors unless you remove `pub_date` - you get:
> 'pub_date' cannot be specified for Newsletter model form as it is a non-editable field. Check fields/fieldsets/exclude attributes of class NewsletterAdmin.
otherwise

Reply:
You're quite right. I forgot to add it to the readonly list